(window.webpackJsonp=window.webpackJsonp||[]).push([[7],{KrR7:function(e,i,t){"use strict";t.r(i);var o=t("KHd+"),n=t("UQSp"),a=t("Kw5r");function r(e){return(r="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e})(e)}a.a.config.optionMergeStrategies;var s={VueRemarkRoot:n.a},c=function(e){var i=e.options.components=e.options.components||{},t=e.options.computed=e.options.computed||{};Object.keys(s).forEach((function(e){"object"===r(s[e])&&"function"==typeof s[e].render||"function"==typeof s[e]&&"function"==typeof s[e].options.render?i[e]=s[e]:t[e]=function(){return s[e]}}))},l=a.a.config.optionMergeStrategies,u="__vueRemarkFrontMatter",f={excerpt:"Protein structure prediction is a critical issue in the field of life sciences, which is of great significance for understanding protein function and various life activities. Currently, semi-parametric deep-learning solutions such as AlphaFold2 have achieved comparable accuracy to experimental techniques such as cryo-electron microscopy. However, due to the heavy reliance on input homologous sequence information, the model has significant limitations in practical scenarios.",title:"AIRFold：systematic protein structure prediction solution",abbr:"AIRFold",authors:[{name:"Xin Hong",link:"https://bowen-gao.github.io/",affiliation_idx:1},{name:"Hongliang Li",link:"https://scholar.google.com/citations?user=Fk49kawAAAAJ&hl=zh-CN&oi=sra",affiliation_idx:1},{name:"Jingjing Gong",link:"https://scholar.google.com/citations?user=MayCLqYAAAAJ&hl=en",affiliation_idx:1},{name:"Yuxuan Song",link:"https://yuxuansong.com/",affiliation_idx:1},{name:"Yinjun Jia",link:"https://www.semanticscholar.org/author/Yinjun-Jia/145105686",affiliation_idx:2},{name:"Keyue Qiu",link:null,affiliation_idx:1},{name:"Han Tang",link:null,affiliation_idx:1},{name:"Haichuan Tan",link:"https://github.com/thchuan2001",affiliation_idx:1},{name:"Yanyan Lan",link:"https://yanyanlan.com",affiliation_idx:1}],affiliations:["Institute for AI Industry Research, Tsinghua University","School of Life Sciences, Tsinghua University"],links:[{name:"Official website report",link:"https://air.tsinghua.edu.cn/info/1007/1807.htm"},{name:"Interview of QbitAI",link:"https://www.qbitai.com/2022/09/37783.html"},{name:"Open source code",link:"https://github.com/THU-ATOM/AIRFold"}]};var h=function(e){e.options[u]&&(e.options[u]=f),a.a.util.defineReactive(e.options,u,f),e.options.computed=l.computed({$frontmatter:function(){return e.options[u]}},e.options.computed)},d=Object(o.a)({},(function(){var e=this,i=e.$createElement,t=e._self._c||i;return t("VueRemarkRoot",[t("h2",{attrs:{id:"project-overview"}},[t("a",{attrs:{href:"#project-overview","aria-hidden":"true"}},[t("span",{staticClass:"icon icon-link"})]),e._v("Project Overview")]),t("p",[e._v("Protein structure prediction is a critical issue in the field of life sciences, which is of great significance for understanding protein function and various life activities. Currently, semi-parametric deep-learning solutions such as AlphaFold2 have achieved comparable accuracy to experimental techniques such as cryo-electron microscopy. However, due to the heavy reliance on input homologous sequence information, the model has significant limitations in practical scenarios.")]),t("img",{staticClass:"mx-auto",attrs:{src:"/image/airfold_pipeline.svg",width:"720"}}),t("p",[e._v("AIRFold, built on the foundation of AlphaFold2, aims to provide scalable, systematic solutions for the critical issue of protein structure prediction in the field of life sciences. AIRFold's unique Homology Miner module focuses on the mining and extraction of co-evolutionary information, intelligently and automatically extracting, analyzing, and processing the co-evolution information within protein homologous sequences (MSA). In addition, AIRFold offers a systematic structural prediction solution, integrating various leading structural prediction models such as AlphaFold2, RoseTTAFold2, single-sequence structure models like OmegaFold and ESMFold, and ultimately ranking and screening all predicted structures using a model quality estimation (MQE) module. To fully integrate these different modules, we provide a microservices architecture along with user-friendly APIs and a web-based graphical interface, making it convenient for developers and biochemical researchers to use our platform for structural prediction.。")]),t("h2",{attrs:{id:"research-progress"}},[t("a",{attrs:{href:"#research-progress","aria-hidden":"true"}},[t("span",{staticClass:"icon icon-link"})]),e._v("Research progress")]),t("p",[e._v("AIRFold has won the global first place for four consecutive weeks in the authoritative protein structure prediction competition CAMEO. The team has built a fully automated control platform, including modules such as homologous sequence augmentation, homologous sequence selection, feature processing, structure prediction, result analysis, and automatic submission, which is much faster than other teams in terms of system response time. Specifically, in terms of the 'Hard' protein sequence, AIRFold is far ahead of second place.")]),t("img",{staticClass:"mx-auto",attrs:{src:"/image/cameo.png",width:"720"}}),t("p",[e._v("For example, for the protein with PDB number 7TVI, which comes from Cas13bt3, this protein has multiple domains, large conformations, and flexible conformations. The quality of homologous sequences obtained from multiple sequence alignment is relatively low. However, through the screening of homology mining modules, the signal-to-noise ratio of long-range interaction information in high-quality homologous sequences has been improved, and the relationship between multiple domains has been more accurately modeled. The results of AlphaFold2 are significantly better than those of Helical-1 and Helical-2, which mainly recognize crRNA (pink part) domains.")]),t("div",{staticClass:"grid grid-cols-2 grid-flow-col gap-2"},[t("img",{attrs:{src:"/image/20220920-141937.png"}}),t("img",{attrs:{src:"/image/20220920-184728.png"}})]),t("h2",{attrs:{id:"future-of-project"}},[t("a",{attrs:{href:"#future-of-project","aria-hidden":"true"}},[t("span",{staticClass:"icon icon-link"})]),e._v("Future of project")]),t("p",[e._v("The prediction of individual proteins and a deep understanding of coevolutionary information are the foundation for the team's future research on proteins and macromolecular drugs. AIRFold focuses on how protein structure determines its function and how it contributes to the development of drugs and therapies, rather than just the isolated issue of structural prediction.\nTherefore, the team is continuing to explore pharmaceutical related issues such as protein point mutations and multi conformation prediction, and is in close communication with relevant enterprises and research institutions. We look forward to more outstanding researchers joining this emerging interdisciplinary field in the future to further leverage the value of AI.")])])}),[],!1,null,null,null);"function"==typeof c&&c(d),"function"==typeof h&&h(d);i.default=d.exports},UQSp:function(e,i,t){"use strict";i.a={name:"VueRemarkRoot",render:function(e){return e("div",null,this.$slots.default)}}}}]);